api/main.py

```diff
@@ -196,6 +196,11 @@ def _get_node_event_data(operation, node):
     return {
         'op': operation,
         'id': str(node.id),
+        'name': node.name,
+        'path': node.path,
+        'group': node.group,
```

Comment:
Atm no service is using `path` or `group` to filter events.
Maybe we can add them to the event data when they are actually required?

Reply:
I think it's very likely they'll get used as part of some pipeline job conditions and they're very generic.  Typically, some jobs would have the condition that they get run when a particular test result is passing (or failing).  I'm also trying to minimise the number of changes in this area, so my judgment was it was less likely to be removed later than added very soon.

Reply:
Okay